### PR TITLE
Add ANE breaking change for Preview 1

### PR DESCRIPTION
**Pull request by gewarren**
Contributes to #26720 (also need to update API ref).

[Preview](https://review.docs.microsoft.com/en-us/dotnet/core/compatibility/windows-forms/7.0/apis-throw-argumentnullexception?branch=pr-en-us-27854).

### CODE REVIEW

**Comment by RussKie** [comment]
Is there a reason why the menu entry reads differently?
image

File: docs/core/compatibility/7.0.md [hunk 1]
```diff
@@ -24,3 +24,9 @@ If you're migrating an app to .NET 7, the breaking changes listed here might aff
 | Title | Binary compatible | Source compatible | Introduced |
 | - | - | - | - |
 | [Deserialize Version type with leading or trailing whitespace](serialization/7.0/deserialize-version-with-whitespace.md) | ❌ | ✔️ | Preview 1 |
+
+## Windows Forms
+
+| Title | Binary compatible | Source compatible | Introduced |
+| - | - | - | - |
+| [Some APIs throw ArgumentNullException](windows-forms/7.0/apis-throw-argumentnullexception.md) | ❌ | ✔️ | Preview 1 |
```
Comment (RussKie) [inline docs/core/compatibility/7.0.md:32]: Not really related to this PR, but this gives me an OCD...
image

If possible I recommend [center aligning](https://markdown.land/markdown-table#aligning-columns) the images (2nd and 3rd columns).
It'd also be great to "fix" the width of the last three columns too. Unfortunately MD syntax does not provide a way to do this, but looks like there are few possible clever ways of achieving this (e.g. [this](https://stackoverflow.com/questions/36121672/set-table-column-width-via-markdown)).

**Review by RussKie — APPROVED** [review]

File: docs/core/compatibility/windows-forms/7.0/apis-throw-argumentnullexception.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- If your code handles <xref:System.NullReferenceException>, replace or add an additional handler for <xref:System.ArgumentNullException>.
+
+## Affected APIs
+
+The following table lists the affected APIs and specific parameters:
+
+| Method/property | Parameter name |
+|-|-|
+| <xref:System.Windows.Forms.ToolStripArrowRenderEventArgs.%23ctor(System.Drawing.Graphics,System.Windows.Forms.ToolStripItem,System.Drawing.Rectangle,System.Drawing.Color,System.Windows.Forms.ArrowDirection)> | `g` |
+| <xref:System.Windows.Forms.ToolStripItemRenderEventArgs.%23ctor(System.Drawing.Graphics,System.Windows.Forms.ToolStripItem)> | `g` |
```
Comment (RussKie) [inline docs/core/compatibility/windows-forms/7.0/apis-throw-argumentnullexception.md:42]: We had a follow up in https://github.com/dotnet/winforms/pull/6507 to make the other arg non-null as well
Not sure the best way to document this:
```suggestion
| <xref:System.Windows.Forms.ToolStripItemRenderEventArgs.%23ctor(System.Drawing.Graphics,System.Windows.Forms.ToolStripItem)> | `g` or `item` |
```
or
```suggestion
| <xref:System.Windows.Forms.ToolStripItemRenderEventArgs.%23ctor(System.Drawing.Graphics,System.Windows.Forms.ToolStripItem)> | `g` |
| <xref:System.Windows.Forms.ToolStripItemRenderEventArgs.%23ctor(System.Drawing.Graphics,System.Windows.Forms.ToolStripItem)> | `item` |
```

I updated https://github.com/dotnet/docs/issues/26720 as well

File: docs/core/compatibility/7.0.md (hunk 1 above)
Comment (gewarren) [reply docs/core/compatibility/7.0.md]: Good suggestion on the centering. I tried the div tag, but got a linter error so took it out. I think the many &nbsp; is just too ugly in the raw Markdown.

Comment (RussKie) [reply docs/core/compatibility/7.0.md]: > Good suggestion on the centering. I tried the div tag, but got a linter error so took it out. 

Do you know who owns the linter? May be the rules can be tweaked or exclusions added?

> I think the many   is just too ugly in the raw Markdown.

I think we should prioritise the end user experience, but I agree that adding "&nbsp;" may be a non scalable maintenance burden.

File: docs/core/compatibility/windows-forms/7.0/apis-throw-argumentnullexception.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+- If your code handles <xref:System.NullReferenceException>, replace or add an additional handler for <xref:System.ArgumentNullException>.
+
+## Affected APIs
+
+The following table lists the affected APIs and specific parameters:
+
+| Method/property | Parameter name |
+|-|-|
+| <xref:System.Windows.Forms.ToolStripArrowRenderEventArgs.%23ctor(System.Drawing.Graphics,System.Windows.Forms.ToolStripItem,System.Drawing.Rectangle,System.Drawing.Color,System.Windows.Forms.ArrowDirection)> | `g` |
+| <xref:System.Windows.Forms.ToolStripItemRenderEventArgs.%23ctor(System.Drawing.Graphics,System.Windows.Forms.ToolStripItem)> | `g` and `item` |
```
Comment (RussKie) [inline docs/core/compatibility/windows-forms/7.0/apis-throw-argumentnullexception.md:42]: From a programmers perspective this reads as "the exception is thrown if both args are null", where as in reality the exception is thrown if any of the args is null.

**Pull request merged**